src/components/common/GridCellExpand.tsx

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable @typescript-eslint/no-non-null-assertion */
```

Comment:
You can probably fix that inside of the file where the probelm occurs, by type checking in some way

Reply:
https://github.com/podkrepi-bg/frontend/pull/1377/files/28f477a92bae5439cadd5c35f26052b9285bd755#diff-6fec82610d155962e61f92f5aee8ee2c729c116f5ef4c9ef178661ba56e502d6R24

I'm guessing this is the line that is problematic. You can create a check before using it and then return the no-non-null-assertion